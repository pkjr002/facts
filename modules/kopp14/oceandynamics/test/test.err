Traceback (most recent call last):
  File "kopp14_preprocess_oceandynamics.py", line 184, in <module>
    kopp14_preprocess_oceandynamics(args.scenario, args.zostoga_model_dir, args.zos_model_dir, not args.no_drift_corr, args.baseyear, args.pyear_start, args.pyear_end, args.pyear_step, args.locationfile, args.pipeline_id)
  File "kopp14_preprocess_oceandynamics.py", line 120, in kopp14_preprocess_oceandynamics
    sZOS = np.apply_along_axis(nanSmooth, axis=0, arr=ZOS, w=smoothwin)
  File "<__array_function__ internals>", line 6, in apply_along_axis
  File "/home/rk509/.conda/envs/conda-entk/lib/python3.7/site-packages/numpy/lib/shape_base.py", line 402, in apply_along_axis
    buff[ind] = asanyarray(func1d(inarr_view[ind], *args, **kwargs))
  File "kopp14_preprocess_oceandynamics.py", line 117, in nanSmooth
    temp[idx] = Smooth(x[idx], w)
ValueError: shape mismatch: value array of shape (37,) could not be broadcast to indexing result of shape (1,)
Traceback (most recent call last):
  File "kopp14_fit_oceandynamics.py", line 282, in <module>
    kopp14_fit_oceandynamics(args.pipeline_id)
  File "kopp14_fit_oceandynamics.py", line 60, in kopp14_fit_oceandynamics
    my_zos = pickle.load(f)
ValueError: peek of closed file
Traceback (most recent call last):
  File "kopp14_project_oceandynamics.py", line 140, in <module>
    kopp14_project_oceandynamics(args.nsamps, args.seed, args.pipeline_id)
  File "kopp14_project_oceandynamics.py", line 52, in kopp14_project_oceandynamics
    my_fit = pickle.load(f)
ValueError: peek of closed file
Traceback (most recent call last):
  File "kopp14_postprocess_oceandynamics.py", line 230, in <module>
    kopp14_postprocess_oceandynamics(args.nsamps, args.seed, args.pipeline_id)
  File "kopp14_postprocess_oceandynamics.py", line 50, in kopp14_postprocess_oceandynamics
    f = open(infile, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: 'oceandynamics-kopp14-oceandynamics_ZOS.pkl'
ls: cannot access oceandynamics-kopp14-oceandynamics*.nc: No such file or directory
mv: cannot stat ‘oceandynamics-kopp14-oceandynamics*.nc’: No such file or directory
