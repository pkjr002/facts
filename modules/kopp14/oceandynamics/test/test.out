Initiating test in /scratch/rk509/test.1666722419...

Extracting data files...

Executing workflow...
python kopp14_preprocess_oceandynamics.py --pipeline_id oceandynamics-kopp14-oceandynamics --scenario rcp85 --zostoga_model_dir ./SLR_ALL/ --zos_model_dir ./zosfinal/ --baseyear 2005 --pyear_start 2020 --pyear_end 2100 --pyear_step 10 --locationfile test/location_list.lst
python kopp14_fit_oceandynamics.py --pipeline_id oceandynamics-kopp14-oceandynamics
Cannot open ZOS file

python kopp14_project_oceandynamics.py --pipeline_id oceandynamics-kopp14-oceandynamics --nsamps 200 --seed 4321
Cannot open fit file

python kopp14_postprocess_oceandynamics.py --pipeline_id oceandynamics-kopp14-oceandynamics --nsamps 200 --seed 4321
Cannot open infile


Collecting output files...

Skipping clean up...
